weekly.ci.jenkins.io.tf

```diff
@@ -0,0 +1,54 @@
+resource "azurerm_resource_group" "weekly_ci_controller" {
+  name     = "weekly-ci"
+  location = "East US 2"
+}
+
+resource "azurerm_managed_disk" "jenkins_weekly_data" {
+  name                 = "jenkins-weekly-data"
+  location             = azurerm_resource_group.weekly_ci_controller.location
+  resource_group_name  = azurerm_resource_group.weekly_ci_controller.name
+  storage_account_type = "StandardSSD_ZRS"
+  create_option        = "Empty"
+  disk_size_gb         = local.weekly_ci_disk_size
+  tags = {
+    environment = azurerm_resource_group.weekly_ci_controller.name
+  }
+}
+
+resource "kubernetes_persistent_volume" "jenkins_weekly_data" {
+  provider               = kubernetes.publick8s
+  metadata {
+    name = "jenkins-weekly-pv"
+  }
+  spec {
+    capacity = {
+      storage = local.weekly_ci_disk_size
+    }
+    access_modes                     = ["ReadWriteOnce"]
```

Comment:
Nitpick: factorize

Reply:
Plan: 4 to add, 0 to change, 0 to destroy.